lib/model/model.go

```diff
@@ -1546,21 +1546,21 @@ func (m *Model) diskChangeDetected(folderCfg config.FolderConfiguration, files [
 
 		if localChange {
 			events.Default.Log(events.LocalChangeDetected, map[string]string{
-				"lastModifiedBy": file.LastModifiedBy.String(),
 				"folderID":       folderCfg.ID,
 				"label":          folderCfg.Label,
 				"action":         action,
 				"type":           objType,
 				"path":           path,
+				"lastModifiedBy": file.LastModifiedBy.String(),
```

Comment:
Also, I don't think you need to call string, it will get converted automatically.